languages/java11/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,35 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+import java.util.HashMap;
+
+
+// This class defines your algorithm.
+class Algorithm extends AbstractAlgorithm<String, String>{
+
+    // This class member can be defined as anything you wish, and stores loaded state and mutable state between requests.
+     HashMap<String, String> context = new HashMap<>();
+
+    // This apply function defines the primary motive driver of your algorithm. Please ensure that the types defined in
+    // your algorithm are the same as those defined in your class implementation above ^
+
+    public String apply(String input){
```

Comment:
Do we want the basic template to use `String apply(String)`. It seems to me that a very, very small percentage of algorithms use `String` input and output, so the first thing a user has to do is go read the docs to figure out how to use JSON input/output.

Contrast that with my argument around hiding the context and load function from the template: I suspect that the percentage of brand new bare Java algorithms that need context and loading is way lower than the percentage of algos that need to start with some JSON.